docs/charon/charon-cli-reference.md

```diff
@@ -358,4 +358,120 @@ Flags:
       --p2p-relays strings                Comma-separated list of libp2p relay URLs or multiaddrs. (default [https://0.relay.obol.tech])
       --p2p-tcp-address strings           Comma-separated list of listening TCP addresses (ip and port) for libP2P traffic. Empty default doesn't bind to local port therefore only supports outgoing connections.
 ```
-You can also consider adding [alternative public relays](../faq/risks.md) to your cluster by specifying a list of `p2p-relays` in [`charon run`](#run-the-charon-middleware).
\ No newline at end of file
+You can also consider adding [alternative public relays](../faq/risks.md) to your cluster by specifying a list of `p2p-relays` in [`charon run`](#run-the-charon-middleware).
+
+## The `exit` subcommand
+
+The `exit` subcommands allows users to exit a validator currently existing in a Charon cluster.
+
+Considering the distributed nature of a Charon cluster, the `exit` commands enables users to submit their partially signed exit messages to a remote API, which will hold it for them until there are enough to be successfully aggregated in a fully signed exit message.
```

Comment:
"remote API" - it is probably the Obol API, shall we clarify this?

Reply:
Oisin explicitly mentioned "remote API" instead of Obol API when he originally reviewed my PR's wording, so I'd leave it as it is for consistency.